app/metrics.yaml

```diff
@@ -1911,6 +1911,20 @@ bookmarks_management:
       - android-probes@mozilla.com
       - erichards@mozilla.com
     expires: never
+  open_all_in_new_tabs:
+    type: event
+    description: |
+      A user opened a folder of bookmarks at once in new tabs.
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/974
+      - https://github.com/mozilla-mobile/fenix/issues/11404
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/21212
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - android-probes@mozilla.com
+    expires: "2021-12-01"
```

Comment:
This will have to be updated.
Asking @eliserichards on whether this should probe should also be set to never expire.

Reply:
The expiration date was set to 1st December because it was set for others similar functions as in `open_in_private_tabs`. But #21038 done by @mdboom changes the expiration to `never`. I can also switch this to `never` if approved here (or a any desired value). Mean while tests will be failing since 1st is passed.

Reply:
Just to get a confirmation from Elise that it's okay to add her email here also.